### PR TITLE
more names for ships and stars:

### PR DESCRIPTION
**Pull request by st-pa-fo**
- add few more names for ships (mainly mineralogy based)
- add myself as dev ship name
- add few more names for stars (star trek memory alpha)
- alphabetic reordering of some ship names
- reordering some ship names (as per #FIXME)
- correcting some typos in ship names (mainly umlauts of proper names)
- filled in "(this description is missing)" where the test script failed before

### CODE REVIEW

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -18082,17 +18088,20 @@ Darquan
 Darrian
 Deadwood
 Deemi
+Delos
 Delphi
 Demeter
 Deneb
 Denebola
+Denobula
```
Comment (geoffthemedio) [inline default/stringtables/en.txt:18095]: almost a duplicate...

File: default/stringtables/en.txt [hunk 2]
```diff
@@ -19542,27 +19573,68 @@ Willow'''
...
+Jett
 Komatite
+Lapislazuli
 Limestone
+Malachite
 Marble
+Melilith
+Menhir
+Nugget
+Obelisk
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: not a mineral, I think?

File: default/stringtables/en.txt [hunk 3]
```diff
@@ -19542,27 +19573,68 @@ Willow'''
...
 Obsidian
+Onyx
+Opal
+Orthoclase
+Pebble
 Pegmatite
+Petrogenesis
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: ?

File: default/stringtables/en.txt [hunk 4]
```diff
@@ -19542,27 +19573,68 @@ Willow'''
...
 Pumice
+Pyrite
+Pyroxene
+Quartz
 Quartzite
+Rock
+Ruby
 Sandstone
+Saphire
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: Sapphire I think?

File: default/stringtables/en.txt [hunk 5]
```diff
@@ -19870,15 +19967,18 @@ Mira
 Mirach
 Mirak
 Mirzam
+Mithuna
 Mizar
 Mothallah
+Mrigasheerisham
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: a bit long...

Comment (st-pa-fo) [reply default/stringtables/en.txt]: was there before, just reordered alphabetically

File: default/stringtables/en.txt (hunk 4 above)
Comment (st-pa-fo) [reply default/stringtables/en.txt]: maybe. but is a name, so who cares ^^ well i can change or remove it if you want

File: default/stringtables/en.txt (hunk 3 above)
Comment (st-pa-fo) [reply default/stringtables/en.txt]: the process of becoming a stone. to petrify.

File: default/stringtables/en.txt (hunk 2 above)
Comment (st-pa-fo) [reply default/stringtables/en.txt]: but can consist of minerals and is stone-related

File: default/stringtables/en.txt (hunk 1 above)
Comment (st-pa-fo) [reply default/stringtables/en.txt:18095]: just loop the levenshtejn or edit distance .. i'm pretty sure there are other close matches. besides, i seem to recall having added that first name many years beack, together with names like Leibowitz and Ingwaz and others where i can clearly reconstruct what was going through my head trying to think of more names … nevermind the credits, just glad its all still here ^^ and Even Happier: someone really did the greek alphabet thing to expand star names, great stuff!

**Comment by st-pa-fo** [comment]
string test still fails, its those removed PLC_EXPLORATION_RESEARCH labels again.

File: default/stringtables/en.txt (hunk 5 above)
Comment (st-pa-fo) [reply default/stringtables/en.txt]: is the 11-letter-limit (which i read somewhere) hard and binding? indeed, name seems long enough to split in two

Comment (geoffthemedio) [reply default/stringtables/en.txt]: I don't know of any particular limit being document anywhere. I just thought that name looks quite a bit longer, and could be awkward in the UI.

File: default/stringtables/en.txt (hunk 4 above)
Comment (geoffthemedio) [reply default/stringtables/en.txt]: if you're editing anyway, please add a `p`

File: default/stringtables/en.txt (hunk 2 above)
Comment (geoffthemedio) [reply default/stringtables/en.txt]: lots of things can consist of stone. the theme of this list kinds of stone or minerals, not stuff made from it.

File: default/stringtables/en.txt (hunk 3 above)
Comment (geoffthemedio) [reply default/stringtables/en.txt]: doesn't fit, I think

File: default/stringtables/en.txt (hunk 1 above)
Comment (geoffthemedio) [reply default/stringtables/en.txt:18095]: that there are other problems or questionable cases doesn't mean this should be left...

File: default/stringtables/en.txt (hunk 2 above)
Comment (st-pa-fo) [reply default/stringtables/en.txt]: seeing how they are mixed together in the end anyway, i fail to see the point here. but whatever, they can go to the 'others' group

**Comment by st-pa-fo** [comment]
i have made the desired adjustments

File: default/stringtables/en.txt (hunk 3 above)
Comment (st-pa-fo) [reply default/stringtables/en.txt]: shortened to 'Petroglyph' and moved to 'others'

**Comment by geoffthemedio** [comment]
can you rebase on master, combine into one commit, and re-push?

**Comment by st-pa-fo** [comment]
i'll try that.

**Comment by st-pa-fo** [comment]
seems like i could not do it first time round. let me try and make a new PR with just the one commit.

**Comment by geoffthemedio** [comment]
What part of rebasing, combining into one commit, and force pushing is giving you trouble...?

**Pull request closed without merging**